command/agent/check.go

```diff
@@ -6,12 +6,14 @@ import (
 	"log"
 	"net"
 	"net/http"
+	"os"
 	"os/exec"
 	"sync"
 	"syscall"
 	"time"
 
 	"github.com/armon/circbuf"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
Surprised there's no official client but didn't see one. There's another by a person that says they work for Docker that's popular.